.gitmodules

```diff
@@ -17,3 +17,6 @@
 [submodule "tools/uf2"]
 	path = tools/uf2
 	url = https://github.com/Microsoft/uf2.git
+[submodule "atmel-samd/frozen/Adafruit_CircuitPython_NeoPixel"]
+	path = atmel-samd/frozen/Adafruit_CircuitPython_NeoPixel
```

Comment:
Would it make more sense to have these in the top level? They should have an API that works across all ports.